tokio/src/runtime/builder.rs

```diff
@@ -87,6 +87,11 @@ pub struct Builder {
 
     #[cfg(tokio_unstable)]
     pub(super) unhandled_panic: UnhandledPanic,
+
+    /// When true, the multi-threade scheduler LIFO slot should not be used.
+    ///
+    /// This option should only be exposed as unstable.
+    pub(super) disable_lifo_slot: bool,
```

Comment:
Would it make sense to move this above `unhandled_panic` so that we keep the unstable member down the bottom?

Same question in `runtime::config::Config`.